data/glsk-file/src/main/java/com/farao_community/farao/commons/data/glsk_file/UcteGlskSeries.java

```diff
@@ -55,8 +55,11 @@ public UcteGlskSeries(Element element) {
         this.area = nodeElement.getAttribute("v");
         this.ucteBusinessType = ((Element) element.getElementsByTagName("BusinessType").item(0)).getAttribute("v");
         this.timeSeriesID = ((Element) element.getElementsByTagName("TimeSeriesIdentification").item(0)).getAttribute("v");
-        this.shareFactor = Double.parseDouble(((Element) element.getElementsByTagName("BusinessType").item(0)).getAttribute("share"));
-
+        if (!getShare(element).isEmpty()) {
+            this.shareFactor = Double.parseDouble(getShare(element));
+        } else {
+            this.shareFactor = 100;
+        }
         if (Objects.requireNonNull(element).getElementsByTagName("CountryGSK_Block").getLength() > 0) {
```

Comment:
Do not test the emptiness of the result, but prefer using `hasAttribute("share")` method of Element class